BasicLayoutsCodelab/app/src/main/java/com/codelab/basiclayouts/MainActivity.kt

```diff
@@ -137,23 +140,25 @@ fun FavoriteCollectionCard(
     modifier: Modifier = Modifier
 ) {
     Surface(
-        shape = MaterialTheme.shapes.small,
+        shape = MaterialTheme.shapes.medium,
+        color = MaterialTheme.colorScheme.surfaceVariant,
```

Comment:
Why are we customizing this color and not using the default?

Reply:
if we dont explicitly set it then the shape is not applied

Reply:
this changes the color of the card to white